projects/lights_control/inc/lights_signal_fsm.h

```diff
@@ -21,7 +21,7 @@ typedef struct LightsSignalFsm {
 // If sync count is passed as 0, then it will use a non-syncing blinker. Thus not raising any
 // sync events.
 StatusCode lights_signal_fsm_init(LightsSignalFsm *lights_signal_fsm,
-                                  LightsBlinkerDuration const blinker_duration, uint32_t count);
+                                  const LightsBlinkerDuration blinker_duration, uint32_t count);
```

Comment:
This doesn't need to be const it is copied already...

Reply:
oh yeah LOL